api/c/indigo/src/indigo_layout.cpp

```diff
@@ -70,8 +70,8 @@ CEXPORT int indigoLayout(int object)
                 mol->clearBondDirections();
                 try
                 {
-                    mol->stereocenters.markBonds();
-                    mol->allene_stereo.markBonds();
+                    mol->stereocentersMarkBonds();
+                    mol->allene_stereoMarkBonds();
```

Comment:
I suggest using the same naming style, `alleneStereoMarkBonds`

Reply:
fixed